ci/docker/base.gpu.py39.wanda.yaml

```diff
@@ -0,0 +1,27 @@
+name: "oss-ci-base_gpu"
```

Comment:
why is py39 special (and requires a new file), and other python versions are using the old one?

Reply:
I want to reserve the old (the better) name for the mulitpy versions, which I plan to use for all python versions. The py39 version is temporary to avoid a big refactoring (to replace it with multipy version) in this PR.

Reply:
I see. could you add a comment into this file, noting that this file is a copy/branch of an existing file (right?) and it is temporary?

same with the base ml one.